app/ui-react/packages/ui/src/Settings/OAuth/OAuthAppExpanderBody.tsx

```diff
@@ -15,52 +14,59 @@ export interface IOAuthAppExpanderBodyProps {
   i18nAlertDetail: string;
 }
 
-export class OAuthAppExpanderBody extends React.Component<
+export const OAuthAppExpanderBody: React.FC<
   IOAuthAppExpanderBodyProps
-> {
-  constructor(props: IOAuthAppExpanderBodyProps) {
-    super(props);
-  }
-  public render() {
-    return (
-      <>
-        {this.props.showSuccess && (
-          <Row>
-            <Col xs={11}>
-              <Alert type={'success'}>
-                <strong>{this.props.i18nAlertTitle}</strong>&nbsp;
-                {this.props.i18nAlertDetail}
-              </Alert>
-            </Col>
-          </Row>
-        )}
-        <Row>
-          <Col xs={12} md={8}>
-            <Form isHorizontal={true}>{this.props.children}</Form>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12} md={8}>
-            <>
-              <Button
-                data-testid={'o-auth-app-expander-body-save-button'}
-                bsStyle="primary"
-                onClick={this.props.onSave}
-                disabled={this.props.disableSave}
-              >
-                {this.props.i18nSaveButtonText}
-              </Button>{' '}
-              <Button
-                data-testid={'o-auth-app-expander-body-remove-button'}
-                onClick={this.props.onRemove}
-                disabled={this.props.disableRemove}
-              >
-                {this.props.i18nRemoveButtonText}
-              </Button>
-            </>
-          </Col>
-        </Row>
-      </>
-    );
-  }
-}
+> = (
+  {
+    children,
+    disableRemove,
+    disableSave,
+    i18nAlertDetail,
+    i18nAlertTitle,
+    i18nRemoveButtonText,
+    i18nSaveButtonText,
+    onRemove,
+    onSave,
+    showSuccess
+  }) => {
+  return (
+    <>
+      {showSuccess && (
+        <Grid sm={11}>
+          <GridItem>
+            <Alert variant={'success'}
+                   title={i18nAlertTitle}>
+              {i18nAlertDetail}
+            </Alert>
+          </GridItem>
+        </Grid>
+      )}
+      <Grid sm={12} md={8}>
+        <GridItem>
+          <Form isHorizontal={true}>{children}</Form>
+        </GridItem>
+      </Grid>
```

Comment:
I think you could probably combine these two grids and set the colspan to 12 for these?  Either that or I wonder if just using a Stack would work just as well here.